src/component/LoginForm.js

```diff
@@ -9,19 +10,31 @@ class LoginForm extends React.Component {
       password: ''
    };
 
+   componentDidMount() {
+
+      let arr = [];
+      arr = JSON.parse(localStorage.getItem('user'));
+
+      if (arr != null) {
+         this.props.history.push('/homepage');
+      }
+   }
+
    authenticateUser = () => {
       const temp = JSON.parse(localStorage.getItem('user'));
       if (temp['email'] == this.state.email && temp['password'] == this.state.password) {
          alert('successful');
```

Comment:
@RajanTank add this package to show alert https://www.npmjs.com/package/react-toastify in whole app